PowerEditor/installer/nativeLang/explorerContextMenuEntryLocal.ini

```diff
@@ -53,7 +53,7 @@ Edit_with_Notepad++=Edytuj w Notepad++
 Edit_with_Notepad++=Edit with Notepad++
 
 [czech.xml]
-Edit_with_Notepad++=Editovat pomocí Notepad++
+Edit_with_Notepad++=Editovat pomocí Notepadu++
```

Comment:
In Croatian we use a phrase "pomoću Notepada", as you can see here:
https://www.google.com/search?q=%22pomo%C4%87u+Notepada%22

In Czech they use a phrase "pomocí Notepadu", as you can see here:
https://www.google.com/search?q=%22pomoc%C3%AD+Notepadu%22
You can see less results because Czechs also use a phrase "pomocí poznámkového bloku"
https://www.google.com/search?q=%22pomoc%C3%AD+Pozn%C3%A1mkoveho+bloku%22
when they talk about Microsoft's Notepad.

So...
Czech: Editovat pomocí Notepadu++
Croatian: Uredi pomoću Notepada++
is correct.

Reply:
In my opinion the correct wording should be just "Notepad++" as this is a brand name, or?